packages/components/Button/src/FAB/FAB.test.tsx

```diff
@@ -2,6 +2,13 @@ import * as React from 'react';
 import { FAB } from './FAB';
 import { checkRenderConsistency, checkReRender } from '@fluentui-react-native/test-tools';
 import * as renderer from 'react-test-renderer';
+import { SvgIconProps } from '@fluentui-react-native/icon';
+
+const TestSvg = require('../../../../../apps/fluent-tester/src/FluentTester/test-data/test.svg');
```

Comment:
Mm doesn't seem great to depend on a file in a package that depends on this one.